wunderio/next-drupal-starterkit
# Various small improvements re: Next.js build, infra, formatting

## Link to feature environment:

[Drupal](https://feature-improvements.next-drupal-starterkit.dev.wdr.io)
[Next.js](https://feature-improvements-next.next-drupal-starterkit.dev.wdr.io)
[Storybook](https://feature-improvements-storyb18b.next-drupal-starterkit.dev.wdr.io)

## Changes proposed in this PR:

Various small improvements "backported" from other projects that use the starterkit. See commit messages for details.

## How to test:

1. Run the usual `./setup.sh -c` locally, should be no errors
2. Skim contents of each commit
3. Check CircleCI build logs have nothing unexpected
4. Check feature environment works nicely
